specs/README.md

```diff
@@ -0,0 +1,84 @@
+# Project Reunion Specs
+
+This directory contains archived and in-progress spec documents for APIs in
+Project Reunion.
+
+Documents in this directory are used as part of a feature and API review
+for features in development and form the basis for public documentation
+and sample code. Write specs so they can be directly consumed by an
+app developer or document author.
+
+## Please DO:
+
+* Provide feedback by **commenting on active pull requests** tagged with
+    [the "ApiReview" label.](https://github.com/microsoft/ProjectReunion/pulls?q=is%3Apr+is%3Aopen+label%3AApiReview)
+* Provide feedback by **opening [issues](https://github.com/microsoft/ProjectReunion/issues/new/choose)
+   regarding spec documents in the [reviewed](https://github.com/microsoft/ProjectReunion/tree/master/specs/reviewed) folder**.
+   Specs in the `reviewed` folder have been previously reviewed for
+   inclusion in Project Reunion.
+
+## Please DO NOT:
+
+* Provide feedback on spec documents in the `archive` folder since those
+    features have been deprecated or altered substantially.
+* Create unsolicited PRs that add new specs. To propose a new feature,
```

Comment:
Can someone outside the Reunion team in the Microsoft Windows orgs create a _solicited_ spec PR? In other words, if I have made a feature proposal or I have read someone else's proposal and there has been a positive response to it, can I make my own spec for the feature?

If so ([which would follow WinUI's model](https://github.com/Microsoft/microsoft-ui-xaml/blob/master/docs/feature_proposal_process.md)), I recommend we make that clear here and in the contributor-guide; otherwise, I recommend we explicitly rule it out here for now.